hardened-chromium-subresource-filter.spec

```diff
@@ -0,0 +1,65 @@
+Name:      hardened-chromium-subresource-filter
+BuildArch: noarch
+Requires:  hardened-chromium
+Source0:   hardened-chromium-blocklist
+License:   GPL-2.0
+Summary:   Subresource filter for hardened-chromium
+Version:   1.0
+# Automatically generated version number, so that it doesn't need to be incremented manually
+%{lua: print("Release: "..os.time().."\n")}
+
+%description
+Filters used by hardened-chromium to provide adblocking.
+
+%install
+INSTALL_DIR="%{buildroot}%{_sysconfdir}/chromium"
+mkdir -p "$INSTALL_DIR"
+install -m 0644 %{SOURCE0} "$INSTALL_DIR/hardened-chromium-blocklist"
+
+%post
+if [ -d "/home/" ]; then
+	cd /home && USERS=*
+	INSTALL_DIR="%{_sysconfdir}/chromium"
+	for USER in $USERS; do
+		OLD_DIR="/home/$USER/.config/chromium"
+		echo "Checking for '$OLD_DIR'"
+		if [ -d "$OLD_DIR" ]; then
+			if [ -d "$OLD_DIR/Subresource Filter" ]; then
+				echo "Removing '$OLD_DIR/Subresource Filter'"
+				rm -r "$OLD_DIR/Subresource Filter"
+			fi
+			NEW_DIR="$OLD_DIR/Subresource Filter/Unindexed Rules/%{release}.%{version}"
```

Comment:
isn't this a mutable directory?

Reply:
this directory is in the user's homedir, which means the subresource filter is writable by processes running under the same user. This seems undesirable / a security issue. We should put it in a directory owned by root and readable by all users

Reply:
fair point